openmrs/apps/clinical/app.json

```diff
@@ -36,6 +36,8 @@
                 { "locale" : "pt", "css": "offline-language-portuguese-brazil"}
             ]
         },
+        "tabularViewIgnoreHeadingsList": ["display", "uuid", "image", "$$hashKey", "activeVisitUuid", "hasBeenAdmitted", "forwardUrl", "programUuid", "enrollment"],
+        "identifierHeading": ["ID", "Id", "id", "identifier", "DQ_COLUMN_TITLE_ACTION"],
```

Comment:
plural - "IdentifierHeadings". Also I wonder why we have to be case sensitive? "ID" or "Id" or "id" all should be replaced with a single "ID". Adds more work for configuration otherwise.

Reply:
In the UI we are making these texts into pascal cases. But again, this can be a decision that the implementers can take if their search queries have same headers of different casings.
 
I'll replace it with `ID` and also update the config key.